### ISSUE TITLE
In the insertion mode, the three types of punctuation marks in Chinese are displayed abnormally.

### ISSUE THREAD

**Issue by sgrwfwn** [vim/vim#11922]
### Steps to reproduce

In the insertion mode, the three types of punctuation marks in Chinese are displayed abnormally.

Paste the Chinese punctuation mark below to see the problem.

‘’  “”   ……

### Expected behaviour

Please solve this problem, thank you.

### Version of Vim

gvim_9.0.1266

### Environment

Windows10

**Comment by brammool** [comment]
Is this in the GUI, console, Windows terminal?
Does the problem go away if you leave Insert mode?

**Comment by sgrwfwn** [comment]
this is in the gui.


This problem exists in both the insert mode and the command mode.&nbsp;


The display is still abnormal after exiting the insert mode.


I uploaded a screenshot. Please have a look.









------------------&nbsp;原始邮件&nbsp;------------------
发件人:                                                                                                                        "vim/vim"                                                                                    ***@***.***&gt;;
发送时间:&nbsp;2023年2月1日(星期三) 下午3:44
***@***.***&gt;;
***@***.******@***.***&gt;;
主题:&nbsp;Re: [vim/vim] In the insertion mode, the three types of punctuation marks in Chinese are displayed abnormally. (Issue #11922)





 
Is this in the GUI, console, Windows terminal?
 Does the problem go away if you leave Insert mode?

**Comment by chrisbra** [comment]
I don't see a screenshot and please answer if this is gui or terminal. If you are using terminal, what terminal and does it make a difference if you are using a different one?

**Comment by sgrwfwn** [comment]
The screenshot of the attachment is clearer. Please have a look.





------------------ 原始邮件 ------------------
发件人:                                                                                                                        "vim/vim"                                                                                    ***@***.***&gt;;
发送时间:&nbsp;2023年2月1日(星期三) 下午3:44
***@***.***&gt;;
***@***.******@***.***&gt;;
主题:&nbsp;Re: [vim/vim] In the insertion mode, the three types of punctuation marks in Chinese are displayed abnormally. (Issue #11922)





 
Is this in the GUI, console, Windows terminal?
 Does the problem go away if you leave Insert mode?

**Comment by sgrwfwn** [comment]
I use gvim on the win10 system, but other terminals have not used it.





------------------&nbsp;原始邮件&nbsp;------------------
发件人:                                                                                                                        "vim/vim"                                                                                    ***@***.***&gt;;
发送时间:&nbsp;2023年2月1日(星期三) 下午4:38
***@***.***&gt;;
***@***.******@***.***&gt;;
主题:&nbsp;Re: [vim/vim] In the insertion mode, the three types of punctuation marks in Chinese are displayed abnormally. (Issue #11922)





 
I don't see a screenshot and please answer if this is gui or terminal. If you are using terminal, what terminal and does it make a difference if you are using a different one?

**Comment by sgrwfwn** [comment]
If you can't view it on github, please click the link below to view it.


https://www.aliyundrive.com/s/L58YPufMBnV






------------------&nbsp;原始邮件&nbsp;------------------
发件人:                                                                                                                        "vim/vim"                                                                                    ***@***.***&gt;;
发送时间:&nbsp;2023年2月1日(星期三) 下午4:38
***@***.***&gt;;
***@***.******@***.***&gt;;
主题:&nbsp;Re: [vim/vim] In the insertion mode, the three types of punctuation marks in Chinese are displayed abnormally. (Issue #11922)





 
I don't see a screenshot and please answer if this is gui or terminal. If you are using terminal, what terminal and does it make a difference if you are using a different one?

**Comment by chrisbra** [comment]
I don't know how to view the screenshot: 
![image](https://user-images.githubusercontent.com/244927/215995433-22036586-7a59-4f57-bb32-90aae61b6a04.png)

Please upload it here.

**Comment by sgrwfwn** [comment]
![Vim9 bug](https://user-images.githubusercontent.com/54033497/215995925-78649f08-bd38-4875-9eb7-00ea26bd9b26.png)

**Comment by sgrwfwn** [comment]
Yes, this setting has been made and the display is normal. Thank you!

I'm just an amateur, not a programmer. I don't have a deep grasp of vim. I have fed back a non-existent problem. I'm sorry!

**Comment by brammool** [comment]
If there is a difference between Normal mode and Insert mode, perhaps the input method is causing a problem?
Try setting the 'iminsert' option to zero.   You can also try setting 'imdisable'.

**Comment by sgrwfwn** [comment]
okay, thank you.



---Original---
From: "Bram ***@***.***&gt;
Date: Wed, Feb 1, 2023 20:32 PM
To: ***@***.***&gt;;
Cc: ***@***.******@***.***&gt;;
Subject: Re: [vim/vim] In the insertion mode, the three types of punctuationmarks in Chinese are displayed abnormally. (Issue #11922)




 
If there is a difference between Normal mode and Insert mode, perhaps the input method is causing a problem?
 Try setting the 'iminsert' option to zero.   You can also try setting 'imdisable'.

**Comment by mao-yining** [comment]
@sgrwfwn it seems that the issue was solved. Please close this issue
